yarn.lock

```diff
@@ -3611,7 +3611,7 @@ debug@^3.1.0, debug@^3.2.7:
   dependencies:
     ms "^2.1.1"
 
-debuglog@*, debuglog@^1.0.1:
+debuglog@^1.0.1:
```

Comment:
The change doesn't require yarn.lock file change, might be a bi-product of the CSS build? Normally I'd reset this.

Reply:
May as commit it otherwise it'll just show up in future PRs